specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json

```diff
@@ -189,6 +142,14 @@
         },
         "description": "Restores all key materials using the SAS token pointing to a previously stored Azure Blob storage backup folder",
         "parameters": [
+          {
+            "name": "storageAuthType",
+            "in": "query",
+            "description": "Type of authentication to use",
+            "required": false,
+            "type": "string",
+            "enum": [ "sasToken", "managedIdentity" ]
+          },
```

Comment:
Why is this needed when you already know the job details from the given job ID? Normally how these endpoints are exposed won't lend themselves to be passed another parameter by our pollers.

Reply:
This is the restore request when called 202 and as a response returns the polling uri. We only need this query parameter for backup (/backup), restore(/restore) and selective key restore (/keys/{keyName}/restore). 

The polling uri endpoints as (/backup/{jobId}/pending), (/restore/{jobId}/pending). We have not added the query parameters to these as they are not needed as you mentioned heath.